README.md

```diff
@@ -39,7 +40,7 @@ Currently the following endpoints are supported:
 - [x] [Team Memberships](https://www.terraform.io/docs/enterprise/api/team-members.html)
 - [x] [Team Tokens](https://www.terraform.io/docs/enterprise/api/team-tokens.html)
 - [x] [Teams](https://www.terraform.io/docs/enterprise/api/teams.html)
-- [x] [Variables](https://www.terraform.io/docs/enterprise/api/variables.html)
+- [x] [Workspace Variables](https://www.terraform.io/docs/enterprise/api/workspace-variables.html)
```

Comment:
Ah, they really changed the concept behind this in the API. Nice... Guess it should be ordered below `Workspace` though 😉

Reply:
I was simply following the alphabetical ordering. Plural `Workspaces` goes after `Workspace Variables` the same as `Teams` went after `Team ...`.

Reply:
My bad, your right 👍